pkg/config/config.go

```diff
@@ -36,13 +36,25 @@ func New(filename string) (*Config, error) {
 
 	fp, err := os.Open(filename)
 	if err != nil {
-		return nil, fmt.Errorf("could not open config file: %v", err)
+		return nil, fmt.Errorf("could not open cfg file: %v", err)
 	}
-	defer fp.Close()
+	defer func() {
+		if err := fp.Close(); err != nil {
+			_, _ = fmt.Fprintf(os.Stderr, "could not close file: %v", err)
```

Comment:
keep it just `fmt.Fprintf(os.Stderr, "could not close file: %v", err)`

Reply:
that would make linters unhappy.

Reply:
I think we can live with unhappy linter here

Reply:
or we should fix that as well; just not as part of this PR :-)

Reply:
We'll fix that in the next PR, as for now I'd keep it code homogeneous and remove `_, _ = ` from this PR